analyzer/pattern_limits.go

```diff
@@ -18,4 +18,8 @@ var patternLimits = map[string]limits{
 		OptimalLimit: 0.95,
 		ApproveLimit: 0.85,
 	},
+	"concepts-basicstrings": {
+		OptimalLimit: 0.99,
+		ApproveLimit: 0.9,
+	},
```

Comment:
Maybe add a step about the pattern-limits above... or even better: we should make a default for the pattern limits if there is no entry here for a specific exercise. I'd set it to default 0.99/0.9 as you did here.

Note: these values define the percentage of similarity to one of the patterns that is still acceptable to be optimal/approved.